daphne/dapf/src/bin/dapf.rs

```diff
@@ -74,7 +74,39 @@ enum Action {
     },
 }
 
-fn main() -> Result<()> {
+#[derive(Debug, Deserialize)]
```

Comment:
lines 77 to 107 appear to have been moved without actually chnaging the code. Can you move this stuff back to where it was?

Reply:
Oops, I totally forgot about this. The file structure should be the way it used to be now.